Makefile

```diff
@@ -39,4 +39,12 @@ release:
 puml:
 	go-plantuml generate -rd . -o go-mms.puml
 
+test_integration: build_mmsd
+	$(eval MMSD_KEY=$(shell ./mmsd keys --gen|awk 'NR==1{print $$3}'))
```

Comment:
This populates the default `state.db` with keys, maybe some tmp `state.db` would be better?

Reply:
Good point, that would be better. I guess then, you would check in that file, and start mmsd with ```./mmsd -w ./test_dir```, or something like that?

Reply:
Why is the DB in the repo now?

Reply:
Ah, I see. It's for tests, right? If so, that may clutter the initial config space of a new instance of mmsd. It would probably be better to put it in a folder for test files, or generate it for the test as I think @mpejcoch suggests?

Reply:
This was what I meant, yes. Generate it on start, but give it a `-w <temp_test_dir>` so that it doesn't interact with whatever might be present from before. It is perhaps less of a problem when the testing is going to be done in a fresh container, so far we need to count on this being built and deployed on the PPI nodes too.

I see an advantage of having the DB in the repo though, which is that it would catch possible regressions towards the DB structure, as the state.db is something that users will keep after upgrading to a new mmsd version.

Reply:
But a populated db will cause conflicts if it is run from a git clone, so it's a bit risky to version control it.

Reply:
Having a committed db isn't in itself a problem in my opinion, but it shouldn't be located at the default location in the repo, but rather in a resource folder for tests that mmsd is pointed to. Then there is no risk of conflicting with a live db.

Reply:
Aha! I misunderstood Martin to begin with. But it simplifies things, to have a static test db in repo, to avoid having to dynamically set api keys for the integrations tests etc. 

So, maybe its ok to create have in ```./test_data/state.db```?

Reply:
Yes, that sounds like a good solution.